webClient/src/app/activity-inprogress-table/activity-inprogress-table.component.scss

```diff
@@ -0,0 +1,28 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+// :host ::ng-deep {
+//     @import "../../../node_modules/carbon-components/scss/globals/scss/styles.scss";
+//   }
+
+#connect-button {
+    display: block;
+    width: 100%;
+}
+
+#cancel-button {
```

Comment:
We now have a standardized format for adding IDs to elements, especially interactables ones. This is new and is used by almost no one except the Zowe Desktop but now we can begin using it here too!

https://github.com/zowe/zlux/wiki/Adding-IDs-to-UI-elements-for-QA-Test-Automation

Reply:
done